app/routes/dump.py

```diff
@@ -50,8 +50,17 @@ def dump_routing(schema, questionnaire_store):
         questionnaire_store.progress_store,
         questionnaire_store.metadata,
     )
+    full_routing_path = router.full_routing_path()
+    response = []
 
-    response = {"routing_path": router.full_routing_path()}
+    for routing_path in full_routing_path:
+        response.append(
+            {
+                "section_id": routing_path.section_id,
+                "list_item_id": routing_path.list_item_id,
+                "routing_path": list(routing_path),
+            }
+        )
```

Comment:
```Python
response = [{
    "section_id": routing_path.section_id,
    "list_item_id": routing_path.list_item_id,
    "routing_path": routing_path.values
} for routing_path in full_routing_path]
```

Notice the values is no longer a protected attribute. Also, I don't think json.dumps needs the routing_path value to be a list, it will handle sets appropriately (i.e. transform them to JSON arrays)